src/pkg/util/oci/helper.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package oci
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sylabs/sif/pkg/sif"
+)
+
+// LoadConfigSpec loads the config.json oci runtime spec
+// from the provided path to a SIF.
+func LoadConfigSpec(Path string) (spec *specs.Spec, err error) {
+	configJSON, err := os.Open(Path)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("JSON specification file %s not found", Path)
+		}
+		return nil, err
+	}
+	defer configJSON.Close()
+
+	// Load the SIF
+	fimg, err := sif.LoadContainerFp(configJSON, true)
+	if err != nil {
+		return nil, fmt.Errorf("while loading SIF file: %s", err)
+	}
+	defer fimg.UnloadContainer()
+
+	name := "oci-runtime-spec"
+	var bn [128]byte
```

Comment:
I *think* it's possible to just do `bn := []byte("oci-runtime-spec")`

I'm also not sure that `bn` is the best name for this, is there something slightly more descriptive that can be used?